helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -579,6 +580,11 @@ public void updateDeploymentGroupHosts(final String groupName, final List<String
 
       client.transaction(ops);
       emitEvents(DEPLOYMENT_GROUP_EVENTS_KAFKA_TOPIC, events);
+    } catch (BadVersionException e) {
+      // some other master beat us in processing this host update. not exceptional.
+      // ideally we would check the path in the exception, but curator doesn't provide a path
+      // for exceptions thrown as part of a transaction.
+      log.debug("error saving rolling updateDeploymentGroupHosts operations: {}", e);
```

Comment:
a debug log will never really be seen - since we log at info above that we are starting some transaction with a list of operations, it would be good to have a matching log entry also at info about how the transaction was beaten / rolled back / whatever the zookeeper term is